docs/Manpage.md

```diff
@@ -2089,7 +2103,7 @@ example, run `export HOMEBREW_NO_INSECURE_REDIRECT=1` rather than just
   <br>If set, `brew install`, `brew upgrade` and `brew reinstall` will never automatically cleanup installed/upgraded/reinstalled formulae or all formulae every `HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS` days.
 
 - `HOMEBREW_NO_INSTALL_UPGRADE`
-  <br>If set, `brew install` will not automatically upgrade installed but outdated formulae
+  <br>If set, `brew install *`formula`*` will not upgrade `*`formula` if it is installed but outdated, disabling the succesfulyindepotent exit code of `brew install <formula`*`.
```

Comment:
Is `succesfulyindepotent` a typo?

Reply:
Yes. https://github.com/Homebrew/brew/pull/12413#discussion_r747388327

Incidentally, I would avoid using "idempotent" here because I would argue that `brew install` is not idempotent by default, but setting `HOMEBREW_NO_INSTALL_UPGRADE` makes it idempotent.

Reply:
Hope it's okay I chime in here.

@carlocab Good point. It is my understanding that setting `HOMEBREW_NO_AUTO_UPDATE` makes `brew install` idempotent because homebrew-core does not change unless the user runs `brew update`. I am not sure that setting `HOMEBREW_NO_INSTALL_UPGRADE` alone provides this guarantee.